src/test/java/sk/stuba/fei/uim/vsa/pr1/bonus/PageableTest.java

```diff
@@ -113,7 +113,7 @@ void shouldGetPageOfStudents() throws NoSuchFieldException, IllegalAccessExcepti
         students = pagedThesisService.findStudents(Optional.empty(), Optional.of("3"), pageable);
         assertNotNull(students);
         assertNotNull(students.getContent());
-        assertEquals(5, students.getContent().size());
+        assertEquals(0, students.getContent().size());
```

Comment:
Toto nie je dobre. Z popisu PR je zrejmé, že metódu `Pageable.next()` máš zrejme zle implementovanú nakoľko tá metóda má vrátiť novú inštanciu Pageable nie upraviť pôvodnú. Takže posledné volanie `findStudents()` pracuje práve s `Pageable(0, 5)`.